docs(observability): how to activate cockpit MTA-2314 (#920)
scaleway/docs-content: observability/cockpit/how-to/activate-cockpit.mdx

---
meta: How to activate your Cockpit
  title: How to activate your Cockpit
  description: This page shows you how to activate your Cockpit
content:
  h1: How to activate your Cockpit
  paragraph: This page shows you how to activate your Cockpit
categories:
  - observability
dates:
  validation: 2022-10-30
  posted: 2022-10-30
---

This page shows you how to activate and monitor your Scaleway products from your Cockpit using the [Scaleway console](https://console.scaleway.com/).

<Message type="requirement">

 You have an account and are logged into the [Scaleway console](https://console.scaleway.com)

</Message>

> # How to activate your Cockpit

Author: ```suggestion
```

> 1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.

Reviewer: ```suggestion
1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

---
meta: How to activate your Cockpit
  title: How to activate your Cockpit
  description: This page shows you how to activate your Cockpit
content:
  h1: How to activate your Cockpit
  paragraph: This page shows you how to activate your Cockpit
categories:
  - observability
dates:

>   validation: 2022-10-17

Reviewer 2: ```suggestion
  validation: 2022-10-30
```

make these dates unique :)

  posted: 2022-10-30
---

> This page shows you how to activate your Scaleway Cockpit using the [Scaleway console](https://console.scaleway.com/), to start monitoring your Scaleway products.

Reviewer 2: ```suggestion
This page shows you how to activate and monitor your Scaleway products from your Cockpit using the [Scaleway console](https://console.scaleway.com/).
```
maybe like this to reduce the number of SCW in the sentence?

<Message type="requirement">

 You have an account and are logged into the [Scaleway console](https://console.scaleway.com)

</Message>

# How to activate your Cockpit

> 1. Log into your [Scaleway console](https://console.scaleway.com/) account.

Reviewer: Unnecessary step I think

> 1. Log into your [Scaleway console](https://console.scaleway.com/) account.

Author: ```suggestion
```

> 1. Log into your [Scaleway Console](https://console.scaleway.com/) account.

Reviewer 2: ```suggestion
1. Log into your [Scaleway console](https://console.scaleway.com/) account.
```

> 2. Click **Cockpit** in the side menu.

Author: ```suggestion
1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

> 3. Click **Activate** to activate your Cockpit. Your Cockpit will be ready in a matter of minutes.

Author: ```suggestion
2. Click **Activate** to activate your Cockpit. Your Cockpit will be ready in a matter of minutes.
```
